docs/source/install-osx.rst

```diff
@@ -0,0 +1,49 @@
+.. _install-osx:
```

Comment:
Ideally this file would be renamed and the bookmark would also be renamed from 'osx' to macos'.